godtools/App/Share/LanguageCodes.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  LanguageCodes.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 7/25/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+
+enum LanguageCodes {
```

Comment:
This is definitely needed.  I know there are some other places in the app where we have hard-coded language codes which should pull from here.

Couple things:
- Can you use a ```struct``` or ```class``` instead of an enum since we aren't using cases.

- Can you make those constants lowerCamelCase to mimic swift conventions. 

https://www.swift.org/documentation/api-design-guidelines/#general-conventions

Reply:
Hey @rachaelblue update here, I think enum is okay here and has the added benefit of not being instantiated. 

Reading through this StackOverflow post here -> https://stackoverflow.com/questions/24244326/swift-global-constant-naming-convention/38078903#38078903

Reply:
Yeah, I use `enum` when I essentially want a namespace and not something instantiable.  But I also see the perspective that that's not what `enum` was designed for.  It would be nice for Swift to have something separate specifically for namespaces.